### ISSUE TITLE
Warnings from yarn regarding node-uuid

### ISSUE THREAD

**Issue by fungilation** [Microsoft/code-push#419]
On every `yarn upgrade`:
```
warning react-native-code-push > code-push > node-uuid@1.4.7: use uuid module instead
warning react-native-code-push > xcode > node-uuid@1.4.7: use uuid module instead
```

Perhaps the warning can be fixed as suggested, if it's "correct"?

Using `code-push` 1.8.0-beta

**Comment by max-mironov** [comment]
Looks like the package `node-uuid` package on npm has been deprecated but the repo hasn't been because its now serving up the source code for `uuid`. More details here https://github.com/kelektiv/node-uuid/issues/155

@silhouettes - I've searched through code-push project and it seems that we are not using `node-uuid` module any more at this time. What do you think, is it ok if we remove this dependency completely from our code-push `package.json`?

**Comment by silhouettes** [comment]
@max-mironov great catch! yes, please remove this dependency completely.

**Comment by silhouettes** [comment]
Oh, @max-mironov - the xcode dependency also uses node-uuid, so the warning will still show up even after my PR is merged. You might need to look at that separately.

**Comment by max-mironov** [comment]
@silhouettes - I've looked closer at this and it turns out that nothing we can do with this warning. 
As it is stated here  https://github.com/react-community/create-react-native-app/issues/86 this warning appears due to our `code-push-react-native` (as well as `react-native`) have dependency upon `xcode `package which is in its turn dependent upon `node-uuid` package. 
So I believe nothing we can do here until `xcode` package will update their dependencies.

**Comment by silhouettes** [comment]
@max-mironov - gotcha! Thanks for looking into that. So I guess we've done all we can on our end.

**Comment by max-mironov** [comment]
@fungilation thank you for referencing the issue in `node-xcode` repository. Currently we'll left this item open and wait for new release of `xcode` to bump dependency accordingly.

**Comment by fungilation** [comment]
it's version bumped to include the move to uuid: https://github.com/alunny/node-xcode/pull/117#issuecomment-290856044

**Comment by max-mironov** [comment]
Hey @fungilation, thanks for heads up!
We have updated xcode dependency on our side [in this PR](https://github.com/Microsoft/react-native-code-push/pull/776).

However currently we will still see this warnings for 2 reasons:
1) `react-native` package depends upon old xcode package, we cannot force them to update it immediately
2) our `react-native-code-push` package have a dependency for `code-push 1.8.0-beta` which is outdated. Unfortunately we should wait for new  `code-push` release which will include [this PR](https://github.com/Microsoft/code-push/pull/422) and after that we can update this dependency also.

We'll try to update this as soon as possible.

**Comment by max-mironov** [comment]
Closing this as fix was released with React Native Code Push 2.0.3.
